store/tikv/2pc.go

```diff
@@ -809,6 +811,15 @@ func getTxnPriority(txn *tikvTxn) pb.CommandPri {
 	if pri := txn.us.GetOption(kv.Priority); pri != nil {
```

Comment:
Did we SetOption() when the `variable.ForcePriority` exists? If so, the old code should also work.